.gitignore

```diff
@@ -8,6 +8,9 @@ bin/
 *.war
 *.ear
 
+# Test folder for conversation
+test
```

Comment:
I don't think we should ignore the test folder. If you want it excluded from the deploy .npmignore is a good option